core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -2282,4 +2282,24 @@ public int getStaleJobThresholdInDays() {
         return Integer.parseInt(getOptional("kylin.tool.health-check.stale-job-threshold-days", "30"));
     }
 
+    public String[] getAssignedServers() {
```

Comment:
Please add comment to describe when and how to use this config.

Reply:
It's advised to add this config in the part for "kylin.cube.*"